README.md

```diff
@@ -103,13 +102,31 @@ pnpm run build
 
 ## Dependencies
 
-- @elizaos/core: workspace:*
+- `@elizaos/core: workspace:*`
 
 ## Future Enhancements
 - Conversation History & Agent Context Backup
 - Cross Agent Data Sharing
 - Encryption with LIT Protocol
 
+## Storacha Client
+
+You can access the Storacha Client directly from the agent code if the plugin is enabled in the `character` file e.g.:
+
+**Upload**
+```js
+const storageClient = await getStorageClient(runtime);
+if (storageClient) {
+    const cid = await storageClient.getStorage().uploadFile(...);
```

Comment:
We could link to docs for the client?

Reply:
Good idea. I will change that. Thx

Reply:
Just updated the docs here and linked to the ElizaOS section in https://github.com/storacha/docs/pull/68